src/crawlee/storages/dataset.py

```diff
@@ -189,6 +190,32 @@ async def get_data(self, **kwargs: Unpack[GetDataKwargs]) -> DatasetItemsListPag
         # https://github.com/apify/apify-sdk-python/issues/140
         return await self._resource_client.list_items(**kwargs)
 
+    async def write_to(self, content_type: Literal['json', 'csv'], destination: TextIO) -> None:
```

Comment:
Is `write_to` named after the same method in Crawlee JS? Why not use `export_data` as on the crawler?

Reply:
No, that's mine. I imagine having `export_data` *and* `export_to` would be confusing. If you have a better name in mind, I'm open to suggestions.